README.MD

```diff
@@ -1,42 +1,258 @@
+  -v teaspeak_certs:/ts/certs \
+  -v teaspeak_config:/ts/config \
+  -v teaspeak_crash_dumps:/ts/crash_dumps \
+  -p 9987:9987 -p 10101:10101/tcp -p 30303:30303/tcp \
+  --restart=unless-stopped --name teaspeak-server teaspeak/server:latest-alpine
 ```
-docker run --rm -i teaspeak:debian -Pgeneral.database.url=sqlite://database/TeaData.sqlite
-``` 
 
-### Ubuntu
-Just build the docker by executing:  
+#### Debian (~427MB, 14 Layers)
+<details>
+    <summary>Build and Run Debian</summary>
+  
+```shell script
+# Build the Image with the latest server
+docker build -f server/debian.Dockerfile -t teaspeak/server:latest-debian ./server
+
+# alternatively with server version <x.y.z>
+# docker build -f server/debian.Dockerfile --build-arg SERVER_VERSION=<x.y.z> -t teaspeak/server:<x.y.z>-debian ./server
+
+# run the Debian based server image
+docker run -d \
+  -v teaspeak_logs:/ts/logs \
+  -v teaspeak_db:/ts/database \
+  -v teaspeak_files:/ts/files \
+  -v teaspeak_certs:/ts/certs \
+  -v teaspeak_config:/ts/config \
+  -v teaspeak_crash_dumps:/ts/crash_dumps \
+  -p 9987:9987 -p 10101:10101/tcp -p 30303:30303/tcp \
+  --restart=unless-stopped --name teaspeak-server teaspeak/server:latest-debian
 ```
-docker build --tag teaspeak:ubuntu server/ubuntu/
-```  
-Running the docker:  
+</details>
+
+#### Ubuntu (~431MB, 17 Layers)
+<details>
+    <summary>Build and Run Debian</summary>
+    
+```shell script
+# Build the Image with the latest server
+docker build -f server/ubuntu.Dockerfile -t teaspeak/server:latest-ubuntu ./server
+
+# alternatively with server version <x.y.z>
+# docker build -f server/ubuntu.Dockerfile --build-arg SERVER_VERSION=<x.y.z> -t teaspeak/server:<x.y.z>-ubuntu ./server
+
+# run the Ubuntu based server image
+docker run -d \
+  -v teaspeak_logs:/ts/logs \
+  -v teaspeak_db:/ts/database \
+  -v teaspeak_files:/ts/files \
+  -v teaspeak_certs:/ts/certs \
+  -v teaspeak_config:/ts/config \
+  -v teaspeak_crash_dumps:/ts/crash_dumps \
+  -p 9987:9987 -p 10101:10101/tcp -p 30303:30303/tcp \
+  --restart=unless-stopped --name teaspeak-server teaspeak/server:latest-ubuntu
 ```
-docker run --rm -i teaspeak:ubuntu
-```  
-Run with parameters:
+</details>
+
+### EnvVars and Build-args (server)
+
+| Variable | Default | Description |
+|:--------:|:-------:|:-----------:|
+|  **TZ**  | `Europe/Berlin` | Sets the timezone within the container. |
+
+The `TZ` variable works everywhere **EXCEPT** in the `alpine` based server image, where the log times are UTC.
+This seems to be a bug with the glib based Alpine Linux image.
+
+(their [Wiki](https://wiki.alpinelinux.org/wiki/Setting_the_timezone) explicitly states that glibc based installs define their timezones differently, yet fail to mention how to set it in that case :confused: :unamused:)
+
+| Build-Arg | Default | Description |
+|:---------:|:-------:|:-----------:|
+| **TEASPEAK_VERSION** | `1.4.12` | This build argument sets the server version to be installed in the resulting image |
```

Comment:
Isn't the build argument called `SERVER_VERSION`?
As well (maybe) add a variable to select the build channel (like `amd64_stable` or `amd64_optimized`).
I would call the variable `SERVER_RELEASE_TYPE` or something like that.

Reply:
I could do that, but as far as I understand your `optimized` branch of releases, they are beta releases?
Based on that I would like to have a separate Dockerfile, which you/I could update separately from the `stable` Dockerfile/Release.
Feel free to correct me and let me know if you prefer a variable over a separate file.